Makefile

```diff
@@ -52,8 +52,8 @@ infra.up: ## Setup localstack for development
 
 testacc:
 	TF_ACC=1 \
-		MASSDRIVER_AWS_ENDPOINT=http://localhost:4566 \
-		MASSDRIVER_EVENT_TOPIC_ARN=${shell make localstack.sns.last.arn} \
+		MASSDRIVER_AWS_ENDPOINT=http://localstack:4566 \
+		MASSDRIVER_EVENT_TOPIC_ARN="arn:aws:sns:us-east-1:000000000000:massdriver-provider-test.fifo" \
```

Comment:
I couldn't get `make localstack.sns.last.arn` to work. I kept getting a pipefail.

```
 $ make localstack.sns.last.arn
parse error: Invalid numeric literal at line 1, column 5
Exception ignored in: <_io.TextIOWrapper name='<stdout>' mode='w' encoding='utf-8'>
BrokenPipeError: [Errno 32] Broken pipe
make[1]: *** [Makefile:76: localstack.sns.list] Error 120
make[1]: write error: stdout
make: *** [Makefile:81: localstack.sns.last.arn] Error 4
```

This resource name is deterministic so I don't think we need a dynamic lookup, though I'm happy to have one if anybody knows how to fix the make command.